CONTRIBUTING.md

```diff
@@ -289,6 +299,32 @@ do not decrease overall code coverage of the codebase - this is aligned with our
 goal to increase coverage over time. Keep track of execution time for your unit
 tests and try to keep them as short as possible.
 
+#### Library Recommendations 
+
+To keep testing practices consistent across the project, it is advised to use this libraries under
+these circumstances:
+
+- For assertions to validate expecations, use `"github.com/stretchr/testify/assert"`
+- For assertions that are required to continue the test, use `"github.com/stretchr/testify/require"`
+- For mocking external resources, use `"github.com/stretchr/testify/mock"`
+- For validating HTTP traffic interactions, `"net/http/httptest"`
+- For generating test data, use `???`
```

Comment:
I have a feeling that there is or should be a library to do this as part of some testing framework.